### PR TITLE
Update Maestro test command to run multiple flows in GH Actions

### PR DESCRIPTION
**Pull request by msasikanth**

### CODE REVIEW

File: .github/workflows/maestro_tests.yml [hunk 1]
```diff
@@ -70,6 +70,6 @@ jobs:
             -PmanifestEndpoint=${{ secrets.MANIFEST_ENDPOINT }}
             :app:installStagingRelease
             
-            maestro test maestroUiFlows/login_flow.yaml
+            ./gradlew maestroTests
```
Comment (Honey14) [inline .github/workflows/maestro_tests.yml:73]: ~Is there a benefit of running via Gradle instead?~ NM, for all future tests. Got it

**Review by Honey14 — CHANGES_REQUESTED** [review]
Looks good, can you update changelog, please?

**Comment by msasikanth** [comment]
> Looks good, can you update changelog, please?

That was intentional. I wanted to discuss this further whether we should include some of the dev changes like dependency updates and CI changes in CHANGELOG or not.

**Review by Honey14 — APPROVED** [review]

**Pull request merged**